doc/classes/NavigationAgent2D.xml

```diff
@@ -229,17 +230,20 @@
 		</signal>
 		<signal name="navigation_finished">
 			<description>
-				Notifies when the final position is reached.
+				Emitted once per loaded path when the agent internal navigation path index reaches the last index of the loaded path array. The agent internal navigation path index can be received with [method get_current_navigation_path_index].
 			</description>
 		</signal>
 		<signal name="path_changed">
 			<description>
-				Notifies when the navigation path changes.
+				Emitted when the agent had to update the loaded path.
```

Comment:
```suggestion
				Emitted when the agent had to update the loaded path:
```

Same for 3D. The points should probably end with a comma or a semicolon (except for the last one), but English punctuation is weird sometimes, so maybe I'm wrong.